pkg/webhook/configuration/validating.go

```diff
@@ -103,6 +104,9 @@ func validateWebhook(hook *configv1alpha1.ResourceInterpreterWebhook, fldPath *f
 	case cc.URL != nil:
 		allErrors = append(allErrors, webhook.ValidateWebhookURL(fldPath.Child("clientConfig").Child("url"), *cc.URL, true)...)
 	case cc.Service != nil:
+		if cc.Service.Port == nil {
+			cc.Service.Port = ptr.To[int32](443)
+		}
```

Comment:
The modification here should be valid, and it maintains consistency with the API definition.
https://github.com/karmada-io/karmada/blob/8d89d77110cf378d09b5f43dcb2e09bb393d61f1/vendor/k8s.io/api/admissionregistration/v1/types.go#L1202-L1205

Can we add a test case to cover this scenario?

Reply:
You are right. Let me add it later.